09_json/nickolee/pkg/puppy/types.go

```diff
@@ -0,0 +1,17 @@
+package puppy
+
+// Puppy data structure stores puppy properties
+type Puppy struct {
```

Comment:
Nice 👍 
I would encourage you to experiment with `omitempty`. This will prevent fields from reaching output if they hold a zero value.
Whether any of these fields actually need it is up to you.

Reply:
Thanks man will play around with it in my golang playground cheers !